multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -856,14 +861,17 @@ def _fit(
             is_distill=teacher_predictor is not None,
         )
 
+        if self._problem_type == NER:
+            self._output_shape += len(OmegaConf.to_object(config.model.ner.special_tags))
+
         config = update_config_by_rules(
             problem_type=self._problem_type,
             config=config,
         )
 
         if self._df_preprocessor is None:
             df_preprocessor = init_df_preprocessor(
-                config=config.data,
```

Comment:
I guess this change can be reversed. Using `config=config.data` should be fine.

Reply:
NerLabelEncoder is initialized init_df_preprocessor, it needs config.model. 

```   
 if label_column is not None and column_types[label_column] == NER_ANNOTATION:
        label_generator = NerLabelEncoder(config)
    else: 
        label_generator = None

    df_preprocessor = MultiModalFeaturePreprocessor(
        config=config.data,
        column_types=column_types,
        label_column=label_column,
        label_generator=label_generator,
    )
```

Reply:
Got it. Thanks.